pkg/windows/windows.go

```diff
@@ -75,8 +75,15 @@ const (
 	serviceNotFound = "status 1060"
 )
 
-// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
-var filesToTransfer map[*payload.FileInfo]string
+var (
+	// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+	filesToTransfer map[*payload.FileInfo]string
+	// InstalledSVCs is a list of Windows services installed by WMCO
+	InstalledSVCs = []string{windowsExporterServiceName, kubeProxyServiceName, hybridOverlayServiceName,
+		kubeletServiceName}
+	// RequiredDirectories is a list of directories to be created by WMCO
+	RequiredDirectories = []string{k8sDir, remoteDir, cniDir, cniConfDir, logDir, kubeProxyLogDir, hybridOverlayLogDir}
```

Comment:
Consider breaking them in multiple lines [as before](https://github.com/openshift/windows-machine-config-operator/pull/498/files#diff-4d8fe3bb66a872f0e4a277f820d60d2efa722f09aa89516be74b98c06b57f3bdL434), for better visibility. 

Same with [InstalledSVCs](https://github.com/openshift/windows-machine-config-operator/pull/498/files#diff-4d8fe3bb66a872f0e4a277f820d60d2efa722f09aa89516be74b98c06b57f3bdR81) .